iOS/TrackbeamTheme/nuget/Xamarin.Themes.TrackBeam.nuspec

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Need to fix up with all the new metadata requirements.

Author, copyright, fwlinks, require accept license